index.html

```diff
@@ -1,21 +1,142 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+
     <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" href="/css/main.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300&display=swap"
+      rel="stylesheet"
+    />
 
     <title>Responsive grid project</title>
-</head>
+  </head>
+
+  <body>
+    <header>
+      <div class="toggle-menu">
+        <hr class="line-one" />
+        <hr class="line-two" />
+        <hr class="line-three" />
+      </div>
+
+      <div class="blue-box"></div>
+      <h1>Maya Nelson</h1>
+      <h2>Project Manager</h2>
+
+      <!-- <ul class="header-nav-menu"> -->
+      <!-- <li class="about-me"><a href=""></a>ABOUT ME</li>
+      <li><a href=""></a>RESUME</li>
+      <li><a href=""></a>PROJECTS</li>
+      <li><a href=""></a>CONTACT</li> -->
+    </header>
+
+    <main>
+      <section class="card">
+        <img
+          src="AdobeStock_230407429.webp"
+          alt="woman-picture"
+          class="profile-img"
+        />
+        <p class="name">Maya<br />Nelson</p>
+        <hr class="card-line" />
+
+        <div class="card-buttons">
+          <button class="card-buttons1">RESUME</button>
```

Comment:
These will link somewhere; so better to use `a`. It can still be styled like a button, but choose `a` if it's going to act like a link.